### PR TITLE
LOG-5481: Write RH lokistack OTEL test

### PR DESCRIPTION
**Pull request by Clee2691**
### Description
This PR adds an E2E test for `lokistack`. This test ensures that `Lokistack` can receive both `OTEL` and  `Viaq` formatted data when spec'd for either one.

/cc @cahartma @vparfonov 
/assign @jcantrill 

### Links
- JIRA: https://issues.redhat.com/browse/LOG-5481

### CODE REVIEW

**Review by jcantrill — COMMENTED** [review]
/hold

File: test/e2e/logforwarding/lokistack/forward_to_lokistack_test.go [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+		// Deploy log generator
+		logGenNS = e2e.CreateTestNamespaceWithPrefix("clo-test-loader")
+		generatorOpt := framework.NewDefaultLogGeneratorOptions()
+		generatorOpt.Count = -1
+		if err = e2e.DeployLogGeneratorWithNamespaceName(logGenNS, "log-generator", generatorOpt); err != nil {
+			Fail(fmt.Sprintf("unable to deploy log generator %v.", err))
+		}
+	})
+
+	It("should send logs to lokistack's OTLP endpoint when dataModel == Otel", func() {
```
[the remark this answers is not in the document]
Comment (Clee2691) [reply test/e2e/logforwarding/lokistack/forward_to_lokistack_test.go:96]: Yes I can do that.

**Pull request merged**